README.md

```diff
@@ -1,16 +1,17 @@
-# Genymotion dev and release keys.
+# Genymotion development and release keys
 
 Copyright (C) Genymobile
 
 ## Description
 
-Repository to store dev and release keys, and corresponding .mk that are included from device.mk.
+This repository stores Genymotion Android development and release keys, and corresponding .mk that are included from device.mk.
```

Comment:
Are .mk useful for end users because it points to vendor/genymotion that we don't provide anywhere?

Reply:
Will do!